### PR TITLE
Feat : Add snackbar for internet connectivity

### LINKED ISSUE #96: Dismiss the loader & show apprpriate message when there is no internet
1. Turn off the internet connection
2. Open travel mate app
3. Just the progress bar is displayed & no error message is displayed.
**Expected : ** Show appropriate snackbar message & dismiss the progress bar.

### PR DESCRIPTION
**Pull request by Pritom14**
Fix #96

### CODE REVIEW

File: Android/app/src/main/java/io/github/project_travel_mate/destinations/CityFragment.java [hunk 1]
```diff
@@ -18,10 +19,13 @@
 import android.widget.AdapterView;
 import android.widget.ArrayAdapter;
 import android.widget.AutoCompleteTextView;
+import android.widget.FrameLayout;
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/destinations/CityFragment.java]: why do we need this?

File: Android/app/src/main/java/io/github/project_travel_mate/destinations/CityFragment.java [hunk 2]
```diff
@@ -18,10 +19,13 @@
...
 import android.widget.ListView;
 import android.widget.ProgressBar;
 import android.widget.Toast;
 
+import com.rey.material.widget.SnackBar;
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/destinations/CityFragment.java]: We don't need this

File: Android/app/src/main/java/io/github/project_travel_mate/destinations/CityFragment.java [hunk 3]
```diff
@@ -81,7 +85,13 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         mHandler    = new Handler(Looper.getMainLooper());
         cityname.setThreshold(1);
 
-        getCity();
+        if (appConnectivity.isOnline()) {
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/destinations/CityFragment.java]: change to `appConnectivity` to `mConnectivity`

**Comment by Swati4star** [comment]
@Pritom14 The changes look good. Squash these commits into single one.

**Comment by Pritom14** [comment]
Can I submit a new PR? 
I am having a lot of error while squashing them

**Comment by Swati4star** [comment]
Sure.

**Pull request closed without merging**